docker/src/main/java/org/jclouds/docker/options/RemoveContainerOptions.java

```diff
@@ -29,6 +29,17 @@ public RemoveContainerOptions force(Boolean force) {
       this.queryParameters.put("force", force.toString());
       return this;
    }
+   
+   /**
+    * Remove the volumes associated to the container
+    * 
+    * @param volume If set to true the volume associated to the container will be removed. 
+    * Otherwise it will not be removed.
+    */
+   public RemoveContainerOptions volume(Boolean volume) {
```

Comment:
Change the parameter to be a primitive as it does not make sense to pass a `null` value here. It could throw a NPE in the next line.

Could you also change the existing methods to use a `boolean` primitive too?